INSTALLATION.md

```diff
@@ -174,6 +174,12 @@ yarn lint:fix
 yarn typecheck
 ```
 
+We are also using the package `Husky` to automatically run the following scripts on your changes whenever you make a pull from your remote repository:(This hook will only run if there are changes in pakage.json file)
```

Comment:
Make two seperate sub sections for pre-commit and post-push hooks explaining what they're doing in detail. Also provide example for both on how to opt out of executing those hooks.